New Project: reveal.js (slides) as proposed in #66

yuwash opened the pull request:
data.json and logo (created from original svg) for reveal.js (slid.es)

yuwash committed:
New Project: reveal.js (slides) as proposed in #66

jancborchardt commented:
@yuwash just some remarks:
- The logo could need some margin around it – because square logos which use the full square size appear a bit big in comparison to the other ones.  Just increase the width and height a bit and leave equal transparent whitespace around.
- The title should probably be »slid.es« to not confuse. Then the description could be »Slides is a presentation web app based on the open source reveal.js framework.«
- We should really add the license and source attribute to every new project. License in this case is MIT, source code is at https://github.com/hakimel/reveal.js

What do you think @libreprojects/contributors?

yuwash committed:
for slides:
data.json: name=slid.es instead of reveal.js, license and source set as suggested at https://github.com/libreprojects/libreprojects/pull/75#issuecomment-30303170
slides.png has margin of 48px each (160×160→256×256)

yuwash commented:
I've already implemented @jancborchardt 's good suggestions. Can some of the @libreprojects/contributors check it again and share your opinions?

yuwash committed:
slides logo scaled to unified format 128×128 in 160×160 canvas

jancborchardt commented:
@yuwash the entry looks good!
